### DISCUSSION TITLE
Addition of flag (-save-pae-json) resulting in output of seperate pae.json files linked to predicted structures

### DISCUSSION THREAD

**Discussion by MHvegholm** [KosinskiLab/AlphaPulldown#248] [Ideas]
The option to output separate pae.json files removes post-processing before custom visualization of pae plots when using tools like [paeviewer](http://www.subtiwiki.uni-goettingen.de/v4/paeViewerDemo).

This feature is already present in alphafold2 as `_save_pae_json_file()` which is [defined](https://github.com/google-deepmind/alphafold/blob/main/run_alphafold.py#L213) and [applied](https://github.com/google-deepmind/alphafold/blob/main/run_alphafold.py#L306-L312) in [`run_alphafold.py`](https://github.com/google-deepmind/alphafold/blob/main/run_alphafold.py)

**Comment by jkosinski** [comment]
Hi @MHvegholm, thanks, we will think about it @DimaMolod just added a feature that saves PAE to the CIF files. We might want to choose that option instead by default. Adding an additional script that creates the jsons based on the saved pkl files is a good idea to increase compatibility with other packages. I created a feature request based on this discussion.
